packages/manager/modules/pci/src/projects/project/kubernetes/components/autoscaling/autoscaling.controller.js

```diff
@@ -0,0 +1,140 @@
+import { AUTOSCALING_LINK } from './autoscaling.constants';
+import { SCALE_DEFAULT_VALUES } from '../../kubernetes.constants';
+
+export default class AutoscalingController {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.ovhSubsidiary = coreConfig.getUser().ovhSubsidiary;
+    this.autoscalingInfoLink = this.getAutoscalingInfoLink();
+  }
+
+  $onInit() {
+    this.setLowestHighestLimit();
```

Comment:
Having trouble understanding as we have both lowest and highest in the name 😅 Maybe we could just name it `setLimits`

Reply:
Yes nice 👍.